src/microbe_stage/ProcessSystem.cs

```diff
@@ -160,6 +160,35 @@ void MakeSureResultExists(Compound compound)
         return result;
     }
 
+    public static float CalculateSpeed(IEnumerable<OrganelleTemplate> organelles, MembraneType membraneType,
```

Comment:
IMO maybe more suitable to be put in the Microbe class as this method pertains to microbe's functioning.

Reply:
Well the Microbe class is already the longest class in the game, so if at all possible, I'd consider putting it somewhere else...

Reply:
Sounds like the `MicrobeSpeciesInfo` class suggested and implemented in #2427 could be used here. It could shorten the `Microbe.cs` file significantly if we switched some info to it. Although, speed depends on duplicated organelles so it can't be moved here.

Reply:
Well this specific calculation *could* as this is the normal movement speed of a species (without any duplicated organelles etc.)

Reply:
This was placed here in order to have the functionality be in a static library. Since the microbe editor makes this calculation with a detached set of organelles, I don't see a way to attach this to the microbe or microbe species class, or any property thereof. I don't love it here, nor am I a big fan of statics, so I'm open to suggestions.

Reply:
I think the suggestion was to still keep it static, but move it to a different class like Microbe or maybe MicrobeSpecies would make a tiny bit more sense (as the MicrobeSpecies uses the method to calculate the speed that it has).

Reply:
You want a static method in the microbespecies class, that takes in a list of organelle definitions unrelated to the organelles the species has, and calculates a base speed for that? That seems like a clear violation of Single Responsibility Principle.

Reply:
Well if you want to make a new helper class with a good name go ahead, but I can personally say that this makes more sense to be in MicrobeSpecies *than* the process system as the movement organelles have *no* relation to processes. So this currently makes even less sense than where I suggested to put this.

Reply:
I feel it would make sense in the `MicrobeSpecies` class as well. But I don't see the necessity for static methods depending on the organelles of the species. Granted, you'd need to create a new species for every mutated case, but as you use it within an encapsulated framework, you could use some garbage collection method C# has to prevent memory overuse. Not optimal, but it makes sense at least.

Other possibility, you could create that method, but now you'd have to pass it organelles changes, that is how many of each type was added/deleted (e.g. as a `Dictionary<OrganelleTemplate, int>`, and then compute with this correction. It could even still give you the base speed of the species if you pass it an empty dict!

Reply:
In auto-evo new species are already being made for every mutation, so that's not the issue. The issue is reusing it with the code in the microbe editor, which has no species to work with at all. Building a new species with every change was what I originally tried to work with in my take on the dynamic MP PR, but you can review that PR for why that became a mess in its own right.